WindowsServerDocs/virtualization/hyper-v/learn-more/Use-local-resources-on-Hyper-V-virtual-machine-with-VMConnect.md

```diff
@@ -89,6 +89,8 @@ You can easily edit your connection settings for VMConnect by running the follow
   
 `VMConnect.exe <ServerName> <VMName> /edit`  
   
+**Note: An elevated command prompt may be required.**  
```

Comment:
Although not used before in this specific document page, the recommended format is a colored Note blurb in the following MarkDown format:
```suggestion
> [!Note]
> An elevated command prompt may be required.
```
Examples : https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/What-s-new-in-Hyper-V-on-Windows#shared-virtual-hard-disks-updated

Reply:
@illfated Thanks
#sign-off